Resources/Private/Fusion/Elements/DatePicker.fusion

```diff
@@ -6,6 +6,8 @@ prototype(Neos.Form:DatePicker) < prototype(Neos.Form.FusionRenderer:FormElement
                 type = 'date'
                 name = ${elementName}
                 value = ${elementValue}
+                value.@process.format = ${Date.format(value, 'Y-m-d')}
+                value.@process.format.@if.isDateTime = ${Type.className(elementValue) == 'DateTime'}
```

Comment:
Shouldn't this instead check against `\DateTimeInterface` (in order to work for instances of `\DateTimeImmutable` and custom implementations, too)?